xcode/Subconscious/Shared/Components/Common/Story/StoryUserView.swift

```diff
@@ -7,6 +7,29 @@
 
 import SwiftUI
 
+/// Adjusts the hit mask of a view to exclude the top-right corner so we can add buttons there
+/// without having to deal with firing both tap targets at once.
```

Comment:
This is inspired by how the same sort of interaction works on Twitter & Bluesky apps.

Reply:
FYI I experimented a lot with nested buttons, it works but you have to hold down on the inner button for ~50-100ms to trigger it, a quick tap always fires the parent handler.

Found many threads telling me not to nest buttons in the first place, but after experimenting on Twitter they have to be doing something similar… either that or the “…” button is floating on top using a ZStack + .offset()